ha/core/event_manager/event_manager.py

```diff
@@ -318,6 +318,19 @@ def get_events(self, component : str) -> list:
             list: List of events.
         """
 
+        key = f'{const.COMPONENT_KEY}/{component}'
+        value = []
+        Log.debug(f"Fetching subscribed events for {key}")
+
+        if self._confstore.key_exists(key):
+            kv = self._confstore.get(key)
+            for k, v in kv.items():
+                if k.endswith(key):
+                    value = json.loads(v)
```

Comment:
value = []
```
value = json.loads(v)
```
assigned again is it append

can we use 
return kv.values()

Reply:
kv.values returns a string and that needs to be converted

Reply:
```
def get_events(self, component : str) -> list:
```
and
```
value = json.loads(v)
```
both have different return type

Reply:
Lets discuss, I am not clear